### ISSUE TITLE
Clear connection issue banner if a good connection comes back

### ISSUE THREAD

**Issue by gordonbrander** [OpenAgInitiative/openag_ui#118]
Or just make it close-able, I suppose?

**Comment by yeliztaneroglu** [comment]
Please Could you give me more information ? Thank you. @gordonbrander

**Comment by gordonbrander** [comment]
@yeliztaneroglu yes!

`scripts/app.js` has a yellow banner that will display any global app errors. One type of error that this banner will show is an error for a dropped HTTP connection.

The banner module is defined in `scripts/common/banner.js`. It can display several kinds of banners, including `AlertDismissable` (can be dismissed by user) and `AlertRefreshable`. Currently HTTP connection errors coming from environment are mapped to AlertRefreshable (in `TagEnvironment`). These should probably be `AlertDismissable`.

Let me know if you want more details. I am happy to coach you through this change. `ARCHITECTURE.md` in the repository should also have some helpful info for how the UI works.

**Comment by yeliztaneroglu** [comment]
How to developing the openag_ui without a working food computer ? @gordonbrander

**Comment by yeliztaneroglu** [comment]
Excuse me @gordonbrander, I wonder if I asked a wrong question ?

**Comment by gordonbrander** [comment]
@yeliztaneroglu no, sorry, just very slow to respond because of US holidays! The easiest way is if you have a Raspberry Pi. The UI can run with Raspberry Pi even if you don't have a Food Computer.

If you don't have a Raspberry Pi, we can get creative and install CouchDB on a laptop, I think (but I haven't tried this approach yet).

**Comment by yeliztaneroglu** [comment]
thank you @gordonbrander , I will buy a Raspberry Pi to myself :) Which Raspberry Pi product do you suggest me to buy ?

**Comment by gordonbrander** [comment]
You'll want a Raspberry Pi 3 and an Arduino Mega.

**Comment by gordonbrander** [comment]
This is actually more difficult than I expected. We need an ID system for alert banners so you can clear a banner by id without accidentally clearing some other banner that was thrown by some other component.

**Comment by gordonbrander** [comment]
I'm creating a workaround for now.
